doc/src/manual/embedding.md

```diff
@@ -268,8 +268,8 @@ JL_GC_POP();
 ```
 
 The `JL_GC_POP` call releases the references established by the previous `JL_GC_PUSH`. Note that
-`JL_GC_PUSH`  is working on the stack, so it must be exactly paired with a `JL_GC_POP` before
-the stack frame is destroyed.
+`JL_GC_PUSH`  is working on the stack, so it must be exactly paired with a `JL_GC_POP` before the
+stack frame is destroyed. This means that both calls must be placed under the same block scope.
```

Comment:
No. The pop can be in a nested scope while the push cannot.

Reply:
Ok!